src/structurizr/view/view_set.py

```diff
@@ -160,9 +167,17 @@ def hydrate(cls, views: ViewSetIO, model: "Model") -> "ViewSet":
             component_views=component_views,
             deployment_views=deployment_views,
             dynamic_views=dynamic_views,
+            filtered_views=filtered_views,
```

Comment:
You could also use the list comprehension from above directly here.

Reply:
Yeah, but I think it'd look messy when Black has wrapped it.